Code/skyrim_ui/src/app/components/server-list/server-list.component.html

```diff
@@ -26,20 +26,20 @@
     />
 
     <div class="filter-checkbox">
-      <app-checkbox
+      <input type="checkbox"
```

Comment:
put it in a new line for consistency
```suggestion
      <input
        type="checkbox"
```